addons/compat_rf/compat_rf_wardrobe/Uniforms.hpp

```diff
@@ -0,0 +1,23 @@
+// CfgWeapons
+
+class Uniform_Base;
+
+// Macros
+#define UNIFORM_BASE(class1,class2)\
+class class1: Uniform_Base { class ace_wardrobe: EGVAR(wardrobe,base_U_sleeves_down) { modifiableTo[] = { QUOTE(class2) }; }; };\
+class class2: Uniform_Base { class ace_wardrobe: EGVAR(wardrobe,base_U_sleeves_up) { modifiableTo[] = { QUOTE(class1) }; }; };
+
+// Simple Variants
+UNIFORM_BASE(U_C_FirefighterFatigues_RF,U_C_FirefighterFatigues_RolledUp_RF)
+UNIFORM_BASE(U_BG_Guerrilla_6_1,U_BG_Guerrilla_RF)
```

Comment:
Readability, and macro usage should add the final `;` when used.

```suggestion
#define UNIFORM_BASE(class1,class2) \
    class class1: Uniform_Base { \
        class ace_wardrobe: EGVAR(wardrobe,base_U_sleeves_down) { \
            modifiableTo[] = {QUOTE(class2)}; \
        }; \
    }; \
    class class2: Uniform_Base { \
        class ace_wardrobe: EGVAR(wardrobe,base_U_sleeves_up) { \
            modifiableTo[] = {QUOTE(class1)}; \
        }; \
    }

// Simple Variants
UNIFORM_BASE(U_C_FirefighterFatigues_RF,U_C_FirefighterFatigues_RolledUp_RF);
UNIFORM_BASE(U_BG_Guerrilla_6_1,U_BG_Guerrilla_RF);
```